jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdConfiguration.java

```diff
@@ -131,7 +131,8 @@ public String getTokenEndpoint()
 
     public void addScopes(String... scopes)
     {
-        Collections.addAll(this.scopes, scopes);
+        if (scopes != null)
+            Collections.addAll(this.scopes, scopes);
     }
 
     public List<String> getScopes()
```

Comment:
This returns mutable data that the client can change. I didn't change that, but it seemed like something to consider changing.